README.md

```diff
@@ -1,13 +1,17 @@
 # Quire
 
-Developed by Getty, Quire is a new, multiformat publishing tool available for immediate access and use. Quire is in limited beta, © J. Paul Getty Trust, and not yet released as open-source software. **For a free license to use Quire, please complete [this form](http://bit.ly/quire-signup).**
+## About Quire 
```

Comment:
I recommend we state at the top and make clear that this is the repo for the Quire website and documentation.